app/src/test/java/de/westnordost/streetcomplete/data/elementfilter/filters/TagOlderThanTest.kt

```diff
@@ -69,11 +69,17 @@ class TagOlderThanTest {
         ), newDate))
     }
 
-    @Test fun `does not match old element with tag only as a prefix`() {
-        assertFalse(c.matches(mapOf("opening_hours:signed" to "tag"), oldDate))
-    }
-
     @Test fun `does not match new element with tag only as a prefix`() {
         assertFalse(c.matches(mapOf("opening_hours:signed" to "tag"), newDate))
     }
+
+    @Test fun `matches any old element without relevant tags`() {
```

Comment:
@westnordost , just to check that this is intended behaviour? It caught me by surprise a bit (especially given the function name), although doesn't impact on the current usages as they are also using a separate element filter.

Reply:
Hmm, I guess this behavior would be unexpected.

Reply:
I think if its fixed #3946 will need some alternative logic for when recycling/cycleway don't have a check date already.

I guess doing `TagOlderThan || ElementOlderThan` would make sense, or `AnyTagOlderThan("recycling:*")`

Reply:
Hmm... right. `check_date:cycleway` could be set but that doesn't mean that `cycleway` is set

Reply:
Besides, the parser does this:

https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/data/elementfilter/ElementFiltersParser.kt#L243

so when written in the element filter expression, it **does** check for the tag being set. Hence, no documentation update in 

https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/data/elementfilter/ElementFilterExpression.kt#L25-L61

is necessary.